.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,32 @@
+# bump-version bumps a new version
+# runs only if it passes all the checks
+name: bump-version
+
+on:
+  workflow_run:
+    workflows: [check-build-test]
+    types: [completed]
```

Comment:
Hey @wxharry, we may also need this:
<img width="409" alt="image" src="https://user-images.githubusercontent.com/32434520/173240236-41e63539-7127-4ee5-a4f0-9a295b24683d.png">

Because:
> Before I show you how we’ll use the new token, I want to bring up two odd quirks that we’ll need to avoid. By default, GitHub seems to avoid running Actions that would be caused by Actions. This seems to work by checking the GITHUB_TOKEN, but since we generated our own it’ll cause each Action push to start a new Action until you suddenly have 1000 commits.
 It seems odd, but to avoid this, we’ll prevent our Action from running if the only files we see changed are the ones our Action itself would change. In this case, only package.json and CHANGELOG.md are ever updated, so we’ll use the [Paths Ignore](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths) feature in GitHub configs to prevent this.

See: https://medium.com/swlh/bump-bump-bump-d0dab616e83

Reply:
Well, "paths-ignore" seems not belong to "workflow_run":

<img width="677" alt="image" src="https://user-images.githubusercontent.com/32434520/173241467-20966b53-6e55-492f-8856-437a2c3d1a97.png">

Reply:
Consider
> Can we put bump-version and check-test-build in one single workflow so that we know with condition to bump a version?
?

Reply:
明天再看了 🤣

Reply:
> Can we put bump-version and check-test-build in one single workflow so that we know with condition to bump a version?

Hey @wxharry, I made things complex and I think your idea is right, I copy your latest (https://github.com/hypertrons/hypertrons-crx/pull/352/commits/1a76cedd5de24bb80b4b1234ddb5d36e8b4074ca) workflow code to a real [repo](https://github.com/tyn1998/workflows-for-hypercrx/) and test it. See the result in another comment.